geowave-grpc-protobuf-shaded/pom.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>mil.nga.giat</groupId>
+  <artifactId>geowave-grpc-protobuf-shaded</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
```

Comment:
also, I think we should version this with geowave versions (just make it use the geowave parent)